lib/visualc/include/strings.h

```diff
@@ -0,0 +1,4 @@
+/*Dummy file to satisfy source file dependencies on Windows platform*/
+#define strcasecmp _stricmp
```

Comment:
It's ok to remove these 2 aliases. They were both replaced in commit 6dca79e5d7be5d8986fcc982b44a4fd91f533906 and won't be coming back, for the reasons mentioned in the commit.